File: src/main/java/br/com/dusty/dkits/util/tag/TagUtils.java
package br.com.dusty.dkits.util.tag;

import br.com.dusty.dkits.gamer.Gamer;
import org.bukkit.entity.Player;

public class TagUtils {
	
	public static void applyTag(Gamer gamer) {
		Player player = gamer.getPlayer();
		
		String displayName = gamer.getRank().format(player.getName());
		
		player.setDisplayName(displayName);
		player.setPlayerListName(displayName);
		//TODO: Tag above head
	}
}

File: src/main/kotlin/br/com/dusty/dkits/util/gamer/TagUtils.kt
package br.com.dusty.dkits.util.gamer

import br.com.dusty.dkits.gamer.Gamer
import org.bukkit.entity.Player

object TagUtils {

	fun applyTag(gamer: Gamer) {
		val player = gamer.player

		val displayName = gamer.rank.format(player.name)

		player.displayName = displayName
		player.playerListName = displayName
		//TODO: Tag above head
	}
}
